Formula/benthos.rb

```diff
@@ -14,15 +14,9 @@ class Benthos < Formula
   depends_on "go" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-
-    src = buildpath/"src/github.com/Jeffail/benthos"
-    src.install buildpath.children
-    src.cd do
-      system "make", "VERSION=#{version}"
-      bin.install "target/bin/benthos"
-      prefix.install_metafiles
-    end
+    system "make", "VERSION=#{version}"
+    bin.install "target/bin/benthos"
+    prefix.install_metafiles
```

Comment:
Not necessary.

https://github.com/Homebrew/brew/blob/a95dd7d48588099fae744cf8970c9e7ea25368bd/Library/Homebrew/build.rb#L159